### PR TITLE
Replaces generic HOS spacesuit with nerfed Mjolnir

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Replaces generic HOS space suit with the old Mjolnir nerfed for modern NSV.

## Why It's Good For The Game
Generic HOS suit is a copy paste from Bee station. This PR would return the Mjolnir which has a pretty legendary history on NSV back into the game in a nerfed version that doesn't protect from fire and has armor about the same as normal security hard suit.

This would replace the hard suit which currently makes you near invincible in space with a lighter still bulky version, which has the option to give you normal running speed after returning from space. So, it would pretty much replace protection for mobility while still keeping the HOS suit slightly better than normal security suit due to the mobility.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

image

image

</details>

## Changelog
:cl:
balance: rebalanced Peacekeeper suit to be usable in normal round and removed thermal insulation
fix: fixed Peacekeeper suit
sprite: replaced old mjolnir sprites with new red ones.
:cl:

### CODE REVIEW

**Comment by SerynEngi** [comment]
I really don't have many comments on the armor itself. The swap of mobility VS tankiness is a sec main thing. I do have a couple comments on the armor values. The large amount of blue also brings to mind the ERT Medic armor.

Energy: Electrical resistance, Specifically against things like tesla shocks and wires. There is no reason to increase this with this armor, I highly recommend knocking this back down to the original value as there isn't a realistic scenario where a HOS would need this.

Bullet: I'd up that to 50 in this particular case. Its a small resistance up, but its not terrible, and makes it better than the boarding armor.

Bomb: I'm not sure that it needed to be nerfed this hard unless regular HOS armor is knocked down this hard. I'd set it to 25 so a random explosion on the ship doesn't down the HOS while he's fighting.

**Comment by BlueHNT** [comment]
> I really don't have many comments on the armor itself. The swap of mobility VS tankiness is a sec main thing. I do have a couple comments on the armor values. The large amount of blue also brings to mind the ERT Medic armor.
> 
> Energy: Electrical resistance, Specifically against things like tesla shocks and wires. There is no reason to increase this with this armor, I highly recommend knocking this back down to the original value as there isn't a realistic scenario where a HOS would need this.
> 
> Bullet: I'd up that to 50 in this particular case. Its a small resistance up, but its not terrible, and makes it better than the boarding armor.
> 
> Bomb: I'm not sure that it needed to be nerfed this hard unless regular HOS armor is knocked down this hard. I'd set it to 25 so a random explosion on the ship doesn't down the HOS while he's fighting.

The armor values are an exact copy of security suit.

File: nsv13/code/modules/clothing/custom_clothes.dm [hunk 1]
```diff
@@ -275,30 +275,26 @@
...
+	hardsuit_type = "peacekeeper_space"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi/peacekeeper
-	armor = list("melee" = 40, "bullet" = 60, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 40, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 15, "energy" = 30, "bomb" = 25, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75, "stamina" = 50)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
-	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
-	var/stomp_cooldown_time = 0.3 SECONDS
+	var/stomp_cooldown_time = 0.5 SECONDS
 	var/current_cooldown = 0
```
Comment (DeltaFire15) [inline nsv13/code/modules/clothing/custom_clothes.dm:299]: Inheritance gore, doublecheck all things that are being inherited from the syndicate hardsuit.

Including, but not limited to:

- Remove ILLEGAL from item_flags
- doublecheck the things in allowed
- syndisuit is weight class normal which fits in backpacks and just about anywhere.
- syndisuit is artifact-proof

I also do not think a ZERO slowdown on-demand hardsuit is good, which the suit toggle gives you in combatmode. Hardsuits that the crew has access to should always have at least some level of drawback. Should be at least 25% or so of base value in my opinion.

**Review by DeltaFire15 — CHANGES_REQUESTED** [review]

File: nsv13/code/modules/clothing/custom_clothes.dm (hunk 1 above)
Comment (BlueHNT) [reply nsv13/code/modules/clothing/custom_clothes.dm:299]: Ah... I haven't checked over the old gore KMC left... my bad

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: nsv13/code/modules/clothing/custom_clothes.dm (hunk 1 above)
Comment (BlueHNT) [reply nsv13/code/modules/clothing/custom_clothes.dm]: @SerynEngi I have pretty much 0 clue how to do any of this you want me to do except change weight class to bulky.

Comment (DuskLight9978) [reply nsv13/code/modules/clothing/custom_clothes.dm]: The suit should have some level of slowdown even in speed form, I also believe the syndie and crew suits should be roughly on par as this is a combat vessel.. they are expected to tussle with the syndicate. Regardless, please add some slow down, less friction to make something weak and buff it than it is to make it strong then nerf it. I'm sure Delta or Bokkie can weigh in at what a reasonable slowdown for a 'speedy' mode hardsuit would be if you ask them.

**Comment by BlueHNT** [comment]
Next issue:

I need people who are aware of the current armor values which tend to be 60 for HOS armors similar to the old hard suit.

What should be the value of the modular hard suit and what should be the slowdown?

Numbers I had in mind would be putting the armor value to 50 and leaving the slowdown be at 0. Making it a bit speedier, at the cost of not being able to stop bullets too well.

Someone (forgot who) said I should make the slowdown around 0.3-0.5 and put it back up to 60 bulletproofing. 

I want some thoughts on this for reference points.

File: nsv13/code/modules/clothing/custom_clothes.dm [hunk 2]
```diff
@@ -275,31 +275,34 @@
...
+	w_class = WEIGHT_CLASS_BULKY
+	item_flags = null
```
Comment (DeltaFire15) [inline nsv13/code/modules/clothing/custom_clothes.dm:301]:
```suggestion
	item_flags = NONE
```

**Comment by DeltaFire15** [comment]
Also doesn't support digitigrade like syndisuit does, so needs 
`supports_variations = NO_VARIATION`

**Comment by BlueHNT** [comment]
> Also doesn't support digitigrade like syndisuit does, so needs `supports_variations = NO_VARIATION`

I have no clue what that is, but I'll add it.

**Comment by DeltaFire15** [comment]
> I have no clue what that is, but I'll add it.

Stuff related to fancy legs.
(Well technically human legs are the weird ones on an evolutionary standpoint, but shh)

**Review by DeltaFire15 — APPROVED** [review]
Code looks fine, balancing up to bokkie.

**Comment by BlueHNT** [comment]
There's a merge issue that I am unaware of. Send help.

**Comment by BlueHNT** [comment]
Updated master.

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**